<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verba Translate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QTextEdit" name="textEdit_2"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
